.bazelrc

```diff
@@ -84,55 +126,103 @@ build:windows --linkopt=/OPT:REF
 build:windows --host_linkopt=/OPT:REF
 build:windows --linkopt=/OPT:ICF
 build:windows --host_linkopt=/OPT:ICF
-build:windows --copt=/d2ReducedOptimizeHugeFunctions
-build:windows --host_copt=/d2ReducedOptimizeHugeFunctions
-build:windows --experimental_strict_action_env=true
```

Comment:
Technically these things are also inherited from TensorFlow, but they set it in `configure.py`.

Reply:
Ah yes, added the reduce huge functions thing to configure.py